src/components/eip3643/Admin.tsx

```diff
@@ -50,20 +50,23 @@ export default function Admin() {
     setTokenSelected(tokenItem || null);
   };
 
-  const tokensData = ownTokens.map((token) => ({
```

Comment:
I always prefer distinct variables and names for better readability and discoverability. This one here has not very telling name, so I would suggest couple of things:
- keep as separate variable, instead of inline in jsx
- rename to smth like `ownTokensData` or `userOwnedTokensData`
- add use memo for small performance benefit